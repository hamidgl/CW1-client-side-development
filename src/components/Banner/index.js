import React from "react";
import "./style.css";

const Banner = () => {
  return <div className="img-wrapper" />;
};

export default Banner;
// banner component to display the banner image
