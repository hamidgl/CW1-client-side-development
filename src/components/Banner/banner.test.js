import * as React from "react";
import { render } from "@testing-library/react";
import Banner from "./";

describe("<Banner />", () => {
  test("should render component", () => {
    const result = render(<Banner />);
    expect(result.container).toMatchSnapshot();
  });
});
