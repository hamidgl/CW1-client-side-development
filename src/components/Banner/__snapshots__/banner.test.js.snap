// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Banner /> should render component 1`] = `
<div>
  <div
    class="img-wrapper"
  />
</div>
`;
